contrib/completions/bash/oc

```diff
@@ -5285,6 +5285,8 @@ _oc_adm_prune_images()
     local_nonpersistent_flags+=("--keep-tag-revisions=")
     flags+=("--keep-younger-than=")
     local_nonpersistent_flags+=("--keep-younger-than=")
+    flags+=("--leave-registry-alone")
+    local_nonpersistent_flags+=("--leave-registry-alone")
```

Comment:
need to regen

Reply:
yep - thanks - will push when make update finishes